src/eradiate/radprops/rad_profile.py

```diff
@@ -367,7 +369,7 @@ def eval_sigma_a_mono(self, w: pint.Quantity) -> pint.Quantity:
         """
         raise NotImplementedError
 
-    def eval_sigma_a_ckd(self, *bindexes: Bindex) -> pint.Quantity:
+    def eval_sigma_a_ckd(self, *bindexes: Bindex, bin_set: str) -> pint.Quantity:
```

Comment:
I overlooked this: for clarity, I rather suggest to name this arg `bin_set_id`.

Reply:
Yeah, that makes sense, I'll do that